CRM/Contact/BAO/Query.php

```diff
@@ -4886,13 +4904,16 @@ public function searchQuery(
     $query = $this->getSearchSQL($offset, $rowCount, $sort, $count, $includeContactIds, $sortByChar, $groupContacts, $additionalWhereClause, $sortOrder, $additionalFromClause, $skipOrderAndLimit);
 
     if ($returnQuery) {
+      \Civi::log()->debug('returnQuery: ' . $query);
```

Comment:
I'm working to put a deprecation in here & call getSearchSQ when return vars are wanted

Reply:
Great, these debug statements are just for the draft PR/work and I don't think any of the "returnQuery=TRUE" queries were causing performance issues (though I don't know because I didn't add timing debug to them) but I wanted to log everything.